internal/test/testdata/cel_eval/hierarchy_funcs.yaml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Do these tests cover something that is not already covered by tests in `cel_lib_test.go`?

Reply:
They are executed through the same code path used by the engine with access to the request object etc. I also use these tests to make sure that the examples work before I add them to the documentation.